src/main/java/net/syscon/elite/api/model/InmateDetail.java

```diff
@@ -126,6 +122,30 @@
     @ApiModelProperty(value = "Country of birth", example = "GBR")
     private String birthCountryCode;
 
+    @ApiModelProperty(value = "In/Out Status", required = true, example = "IN", allowableValues = "IN,OUT")
```

Comment:
Is there some way that the swagger doc can show that these are all not going to be returned unless extra details are specified?  

(In REST terms they would probably be defined as different media types but theres quite a lot of overhead getting that to work and more burden on the client)

Reply:
I see there's some doc on the flags, could maybe add a `note` on affected fields?  🤷‍♂️  not sure how thats represented in the generated swagger documentation